rootfs/usr/share/inputplumber/devices/50-ayn_loki_max.yaml

```
@@ -36,9 +36,11 @@ source_devices:
     iio:
       name: i2c-BMI0160:00
       mount_matrix:
-        x: [1, 0, 0]
-        y: [0, 0, -1]
-        z: [0, 1, 0]
+        # Calculated from observed gyro output
+        # TODO: Verify updated matrix on actual hardware
+        x: [-1, 0, 0]
```

Based on your diagram of positive/negative deflection, these should be inverted.
```suggestion
        x: [1, 0, 0]
```